.make.defaults

```diff
@@ -179,6 +181,18 @@ __check_defined = \
 	-find . -name '*.egg-info' | xargs rm -rf
 	-find . -name '__pycache__' | xargs rm -rf
 	-rm -rf dist
+	-rm -rf $(ARTIFACTS_DIR) > /dev/null 2>&1
+
+.PHONY: .defaults.docker-save-image
+.defaults.docker-save-image:
+	@# Help: Save docker image as tar file to ${ARTIFACTS_DIR} directory.
+	mkdir -p ${ARTIFACTS_DIR}/
```

Comment:
Now that we're trying to add checks, can you add a check for DOCKER_IMAGE_NAME.  Maybe this (and others) goes at the top of this file though instead of at each referencing target?  I guess its the "global" ones like this that could go at the top of this file?

Reply:
Checking `DOCKER_IMAGE_NAME` in the top of this file results in an error as its is set in the Makefiles that generate Docker images and include .make.defauls, for example  in `.make.transforms`. So I added it to `image` and `publish` targets.